### PR TITLE
Release 0.5.0 - Clara

### PR DESCRIPTION
**Pull request by edwinvandenbelt**
Breaking release. Instead of kebab-cased json, we now use camelCased json. Also other breaking issues. Look at the release notes in wiki

### CODE REVIEW

File: TOMP-API.yaml [hunk 1]
```diff
@@ -1681,162 +1521,264 @@ components:
...
+          description: free text
+        asset:
+          $ref: '#/components/schemas/asset'
+
+    legProgress:
       type: object
       description: provides current asset location & duration and distance of the current leg
       required:
-        - coordinates
+        - coordinate
```
Comment (Tjalle) [inline TOMP-API.yaml:1684]: coordinate is a req. field, but in the properties below it's still coordinate**s**

File: TOMP-API.yaml [hunk 2]
```diff
@@ -1681,162 +1521,264 @@ components:
...
- 
+
+    legState:
+      type: string
+      description: status of a leg
+      enum: [NOT_STARTED, PREPARING, IN_USE, PAUSED, FINISHING, FINISHED, ISSUE_REPORTED]
+
+    license:
+      type: object
+      description: driver or usage license for a specific user. Contains the number and the asset-type you're allowed to operate (e.g. driver license for CAR)
```
Comment (Tjalle) [inline TOMP-API.yaml:1700]: and the _assetType_ you're allowed

File: TOMP-API.yaml [hunk 3]
```diff
@@ -1847,87 +1789,154 @@ components:
...
+            radius:
+              description: Maximum distance a user wants to travel to reach asset in metres, e.g. 500 metres
+              type: number
+            to:
+              $ref: '#/components/schemas/coordinates'
+            travellers:
+              description: the amount of people that have to travel from `from` to `to` [https://github.com/efel85/TOMP-API/issues/56]
+              type: number
+            useAssets:
+              description: when you use the /operator/available-assets and you want to book a displayed asset, you must be able to request o a planning-option for the specific asset (with provide-ids=true), post a booking with the provided id and send directly a commit. This field should contain the asset to book.
```
Comment (Tjalle) [inline TOMP-API.yaml:1827]: when you use the /operator/_availableAssets_

File: TOMP-API.yaml [hunk 4]
```diff
@@ -1847,87 +1789,154 @@ components:
...
-                $ref: '#/components/schemas/timestamp'
-              
-    stop-reference:
+
+    stationInformation:
+      type: object
+      required:
+        - stationId
+        - name
+        - coordinate
```
Comment (Tjalle) [inline TOMP-API.yaml:1906]: required/properties-> coordinate vs coordinates

File: TOMP-API.yaml [hunk 5]
```diff
@@ -2402,16 +2372,16 @@ components:
       schema:
         type: string
       description: Version of the API.
-      example: 1.1
+      example: 0.5.0
     maasId:
       in: header
       name: maas-id
```
Comment (Tjalle) [inline TOMP-API.yaml:2378]: _maasId_

File: TOMP-API.yaml [hunk 6]
```diff
@@ -72,17 +72,17 @@ paths:
...
               schema:
-                $ref: '#/components/schemas/planning-options'
+                $ref: '#/components/schemas/planningOptions'
           headers:
             Location:
-              description: The URI where the created options can be found, in case of 'provide-ids' = true
+              description: The URI where the created options can be found, in case of 'provideIds' = true
               schema:
                 type: string
                 example: '/planning-options/1234'
```
[the remark this answers is not in the document]
Comment (edwinvandenbelt) [reply TOMP-API.yaml:88]: Tjalle, we agreed on kebab-casing the URLs.

File: TOMP-API.yaml [hunk 7]
```diff
@@ -754,12 +989,12 @@ paths:
...
               schema:
-                $ref: '#/components/schemas/journal-entry'
+                $ref: '#/components/schemas/journalEntry'
         '400':
           $ref: '#/components/responses/400BadRequest'
         '401':
           $ref: '#/components/responses/401Unauthorized'
- 
+
   /support/{id}/request/{support-type}:
```
[the remark this answers is not in the document]
Comment (edwinvandenbelt) [reply TOMP-API.yaml:998]: And this is a placeholder. Technically it should already be replaced before doing anything with it.

File: TOMP-API.yaml [hunk 8]
```diff
@@ -834,628 +1071,305 @@ paths:
...
+        cardSubType:
+          description: mandatory in case of OTHER. Can be used in bilateral agreements.
           type: string
-        card-description:
+        cardDescription:
           description: description of the card
           type: string
-        card-acceptors:
+        cardAcceptors:
           description: references to maas-ids of accepting parties. Only if applicable (DISCOUNT).
```
[the remark this answers is not in the document]
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1254]: check

File: TOMP-API.yaml (hunk 1 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1684]: Check, I'll repair this one

File: TOMP-API.yaml (hunk 2 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1700]: yes. repaired

File: TOMP-API.yaml (hunk 4 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1906]: repaired

File: TOMP-API.yaml (hunk 5 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:2378]: kebab case => camelCase was only for the JSON.

File: TOMP-API.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:88]: ok, my bad.

File: TOMP-API.yaml [hunk 9]
```diff
@@ -338,14 +342,16 @@ paths:
...
           $ref: '#/components/responses/400BadRequest'
         '401':
           $ref: '#/components/responses/401Unauthorized'
         '404':
           $ref: '#/components/responses/404NotFound'
+        '410':
+          $ref: '#/components/responses/410Gone'
       callbacks:   # webhooks
         # as described in https://swagger.io/docs/specification/callbacks/
         booking-operations:
```
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:357]: not a issue -> url

File: TOMP-API.yaml [hunk 10]
```diff
@@ -611,25 +687,181 @@ paths:
           $ref: '#/components/responses/400BadRequest'
         '401':
           $ref: '#/components/responses/401Unauthorized'
-               
-  /operator/pricing-plans:
+
+  /operator/available-assets:
```
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:691]: not a issue -> url

File: TOMP-API.yaml [hunk 11]
```diff
@@ -611,25 +687,181 @@ paths:
...
+              description: ISO 639-1 two letter language code
+              example: nl
+              schema:
+                type: string
+        '400':
+          $ref: '#/components/responses/400BadRequest'
+        '401':
+          $ref: '#/components/responses/401Unauthorized'
+
+  /operator/pricing-plans:
```
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:844]: not a issue -> url

File: TOMP-API.yaml [hunk 12]
```diff
@@ -713,12 +948,12 @@ paths:
...
                 items:
-                  $ref: '#/components/schemas/journal-entry'
+                  $ref: '#/components/schemas/journalEntry'
         '400':
           $ref: '#/components/responses/400BadRequest'
         '401':
           $ref: '#/components/responses/401Unauthorized'
- 
+
   /payment/{id}/claim-extra-costs:
```
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:957]: not a issue -> url

File: TOMP-API.yaml [hunk 13]
```diff
@@ -834,628 +1071,305 @@ paths:
...
- 
-    booking-state:
-      description: The life-cycle state of the booking (from NEW to FINISHED)
-      type: string
-      enum: [NEW, PENDING, RELEASED, EXPIRED, CONDITIONAL_CONFIRMED, CONFIRMED, CANCELLED, STARTED, FINISHED]
-      example: CONFIRMED
-     
+
+    conditionDeposit:
+      description: in case the TO demands a deposit before usage. Requesting and refunding should be done using the /payment/claim-extra-costs endpoint.
```
[the remark this answers is not in the document]
Comment (Tjalle) [reply TOMP-API.yaml:1314]: not a issue -> url

File: TOMP-API.yaml (hunk 3 above)
Comment (Tjalle) [reply TOMP-API.yaml:1827]: not a issue -> url

**Pull request merged**